### PR TITLE
【paddle.fleet】Fleet run graph in Executor and add two more strategies

### PR DESCRIPTION
**Pull request by guru4elephant**
### PR types

New features

### PR changes

APIs

### Describe

Fleet distributed_optimizer.minimize function may generate graph structure for runtime, and current design make the graph structure as an attribute for Program. When we are calling Executor.run, we can check if the input program has _graph to run. 

`fuse_grad_size_in_MB` and `_fuse_grad_size_in_TFLOPS` are two properties for gradient fusion. 
`fuse_grad_size_in_MB` is the size of  Tensor in MegaBytes that will be fused during backpropagation.
`fuse_grad_size_in_TFLOPS` is the TeraBytes of Computation between two adjacent fusion point.\
Currently, these two properties are not valid and we will make the properties applicable in next couple of PRs.

### CODE REVIEW

File: python/paddle/fleet/base/distributed_strategy.py [hunk 1]
```diff
@@ -378,6 +378,30 @@ def fuse_all_reduce_ops(self, flag):
...
             print("WARNING: fuse_all_reduce_ops should have value of bool type")
 
+    @property
+    def fuse_grad_size_in_MB(self):
+        return self.strategy.fuse_grad_size_in_MB
+
+    @fuse_grad_size_in_MB.setter
+    def fuse_grad_size_in_MB(self, value):
+        if isinstance(value, int):
+            self.strategy.fuse_grad_size_in_MB = value
```
Comment (gongweibao) [inline python/paddle/fleet/base/distributed_strategy.py:388]: Why only int?

File: python/paddle/fleet/base/distributed_strategy.py [hunk 2]
```diff
@@ -378,6 +378,30 @@ def fuse_all_reduce_ops(self, flag):
...
+        else:
+            print("WARNING: fuse_grad_size_in_MB should have value of int type")
+
+    @property
+    def _fuse_grad_size_in_TFLOPS(self):
+        return self.strategy.fuse_grad_size_in_TFLOPS
+
+    @_fuse_grad_size_in_TFLOPS.setter
+    def _fuse_grad_size_in_TFLOPS(self, value):
```
Comment (gongweibao) [inline python/paddle/fleet/base/distributed_strategy.py:397]: Same as above?

File: python/paddle/fleet/base/distributed_strategy.py (hunk 1 above)
Comment (guru4elephant) [reply python/paddle/fleet/base/distributed_strategy.py:388]: most of the models are bigger than MB?

File: python/paddle/fleet/base/distributed_strategy.py (hunk 2 above)
Comment (guru4elephant) [reply python/paddle/fleet/base/distributed_strategy.py:397]: Oh, this should be float, I will fix it.

File: python/paddle/fleet/base/distributed_strategy.py (hunk 1 above)
Comment (gongweibao) [reply python/paddle/fleet/base/distributed_strategy.py:388]: yeah. Float is better.

**Review by zhwesky2010 — APPROVED** [review]
test_fleet_meta_optimizer ==> test_fleet_recompute_meta_optimizer + test_fleet_graph_execution_meta_optimizer + test_fleet_graph_executor, So approve it!

**Pull request merged**